server/tests/readStatistics.test.js

```diff
@@ -0,0 +1,44 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+
+const baseUrl = '/api/v1';
+let userToken;
+
+describe('User statistics test', () => {
+  before((done) => {
+    chai
+      .request(app)
+      .post(`${baseUrl}/users`)
+      .send({
+        user: {
+          firstname: 'User',
+          lastname: 'lastname',
+          username: 'stat_user1',
+          email: 'statistics@mail.com',
+          password: 'P@ssw0rd123'
+        }
+      })
+      .end((err, res) => {
+        const {
+          user: { token }
+        } = res.body;
+        userToken = token;
+        done();
+      });
+  });
+
+  it('should return user read statistics', (done) => {
+    chai
+      .request(app)
+      .get(`${baseUrl}/user/readStat`)
+      .set('Authorization', `Bearer ${userToken}`)
+      .end((err, res) => {
+        const { status } = res.body;
+        expect(status).to.equal(200);
```

Comment:
Great job from you Martins, please improve the test by also checking other properties in the response. For instance, properties to expect in the response body, doing that will make it easy to track any intentional or accidental change made to the response returned.

Reply:
Thanks for the feedback, but the line is totally covered %100 and cannot be improved in any way, did you make sure to pull this branch to test the feature and run test locally at all? Please ensure to do that.

Reply:
Cool